week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,9 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
+  typeof num === "number";
```

Comment:
Test this code with 3.6 (it should return false) because it is a float and it is not on excludedNums